docs/additional-deployment-options/configure_cloudwatch_logs_delivery.md

```diff
@@ -31,16 +31,4 @@ additional_log_groups_to_elk = {
   "HelloWorldTask" = "/aws/lambda/cumulus-example-HelloWorld"
   "MyCustomTask" = "my-custom-task-log-group"
 }
-```
-
-## ESDIS Metrics
-
-For NASA/NGAP deployments an operator should make a request to the metrics team for write access and the correct shared Logs Destination for further processing in their [ELK](https://www.elastic.co/elk-stack) stack.
-
-To be able to access logs sent to the Metrics ELK stack through the `/logs` endpoint, be sure to also configure the metrics variables.
-
-```hcl
-metrics_es_host = "metricshost.cloudfront.net"
-metrics_es_username = "user"
-metrics_es_password = "password"
-```
+```
```

Comment:
This is all in the wiki, so removing from here

Reply:
So the rest of this doc is relevant whether you're on NGAP with cloud metrics or not? For the NGAP users, should we link to that wiki from this page?

Reply:
They should all go through the wiki, which links here.